app/controllers/presentations_controller.rb

```diff
@@ -5,10 +5,12 @@ def show
 
   def new
     @presentation = Presentation.new
+    authorize @presentation
```

Comment:
I'll try to find a nice way to implement this. As I recall a before filter is called before controller method is called.  Since we need to have the instance available to authorize this it might be kind of tricky but I'll dive on in.